src/commands/create.js

```diff
@@ -52,9 +56,58 @@ function create (opts) {
       if (err && err.name === 'ExistsError') return bus.emit('exit:warn', 'Archive already exists.')
       if (err) return bus.emit('exit:error', err)
 
-      // TODO: dat.json creation/write key
-      state.dat = dat
-      bus.emit('dat')
+      // create before import
+      var datjson = DatJson(dat.archive, { file: path.join(opts.dir, 'dat.json') })
+      fs.readFile(path.join(opts.dir, 'dat.json'), 'utf-8', function (err, data) {
+        if (err) return doPrompt()
+        data = JSON.parse(data)
+        debug('read existing dat.json data', data)
+        doPrompt(data)
+      })
+
+      function doPrompt (data) {
+        if (!data) data = {}
+
+        var schema = {
+          properties: {
+            title: {
+              description: chalk.magenta('Dat Title'),
+              default: data.title || '',
+              // pattern: /^[a-zA-Z\s\-]+$/,
+              // message: 'Name must be only letters, spaces, or dashes',
+              required: false
+            },
+            description: {
+              description: chalk.magenta('Dat Description'),
+              default: data.description || ''
+            },
+            doImport: {
+              description: chalk.magenta('Would you like to import your files?'),
+              default: 'yes'
+            }
+          }
+        }
+        prompt.message = chalk.green('> ')
+        prompt.delimiter = '' // chalk.cyan('')
```

Comment:
Unfortunately it adds a delimiter after the `message` and after the description:

![screen shot 2017-05-17 at 10 33 56](https://cloud.githubusercontent.com/assets/684965/26167716/716ad610-3aec-11e7-9901-61abf7baf46c.png)

(In this case, I have `My dataset` in `dat.json` already, so it defaults to that).

I could change the message to something that'd make more sense with a delimiter.